### ISSUE TITLE
long touch on gallery images: offer save picture locally

### ISSUE THREAD

**Issue by ralfhauser** [mmmatthew/water-fountains.org#168]
when on cell-phone, this would be great

2) Also on computer, mouse right-click should offer to save the image locally

**Comment by ralfhauser** [comment]
3) when the image is only black, offer 'reload'

**Comment by mmmatthew** [comment]
1 and 2 relates to the image gallery library and may be fixed with https://github.com/water-fountains/proximap/issues/126

3 is a duplicate of https://github.com/water-fountains/proximap/issues/166 and will be marked off-topic.

**Comment by ralfhauser** [comment]
> 1 and 2 relates to the image gallery library and may be fixed with #126
Confused: there are enhancements while #126 concerns a bug.
4) Or are we behind in library versions and the new version offers "save locally* ?
5) ETA of the fix/redux new version ?

**Comment by mmmatthew** [comment]
both 1 and 2 are "features" that come by default if an image is not hidden by other html elements or overridden. 

4. We are behind because we need the new redux version before we can upgrade. The question is whether we upgrade or revert back to the old library which is not as feature-rich but was more robust: https://lukasz-galka.github.io/ngx-gallery-demo/. In that library, when viewing the images "full screen", you have the option to save as requested.
5. https://github.com/angular-redux/platform/tree/upgrade-angular-7/packages/store does not have much activity so I can't say. Here is wondering if the project is alive: https://github.com/angular-redux/platform/issues/78

**Comment by ralfhauser** [comment]
what are our alternatives to this apathetic redux project ?

**Comment by katjahenke** [comment]
Hi - just trying to figure this thing out...I might have to stick to taking photos! Please let me know if you have receive this before I shar my poem

**Comment by mmmatthew** [comment]
Hi @katjahenke! Yes, your message is visible to all. You can think of issues as discussion. If you think your poem fits in this discussion, feel free to post it here. Otherwise, you can create a new issue.

**Comment by mmmatthew** [comment]
> There's another library to use Redux with Angular, which is https://github.com/ngrx/platform
